web/Views/Property/ViewProperty.cshtml

```diff
@@ -1,132 +1,338 @@
-﻿@using Microsoft.AspNetCore.Mvc.TagHelpers
-@model PropertyViewModel
+﻿@model PropertyDetailsViewModel
 
 @{
     ViewData["Title"] = "View Property";
 }
 
-<div class="container">
-    <div class="row mb-3">
-        <h3>View property</h3>
+@functions
+{
+    string GetPreferenceFont(bool? preference)
+    {
+        return preference ?? false ? "" : "font-orange";
+    }
+
+    string GetPreferenceIcon(bool? preference)
+    {
+        return preference ?? false ? "bi-check-lg" : "bi-x-lg";
+    }
+}
+
+<div class="mt-5 row">
+    <div class="col-auto mb-2">
+        <h1 class="font-h1 d-inline">@Model.Property.Address.ToShortAddressString()</h1>
+    </div>
+    <div class="col-auto mb-2 d-flex align-items-center">
+        @switch (Model.Property.Availability)
+        {
+            case AvailabilityState.Occupied:
+                <span class="status-pill-occupied rounded-pill px-3 py-1">
+                    <i class="bi bi-people me-1"></i>
+                    Occupied
+                </span>
+                break;
+            case AvailabilityState.Unavailable:
+                <span class="status-pill-unavailable rounded-pill px-3 py-1">
+                    <i class="bi bi-x-lg me-1"></i>
+                    Unavailable
+                </span>
+                break;
+            case AvailabilityState.Draft:
+                <span class="status-pill-draft rounded-pill px-3 py-1">
+                    <i class="bi bi-clock me-1"></i>
+                    Draft
+                </span>
+                break;
+            default:
+                <span class="status-pill-live rounded-pill px-3 py-1">
+                    <i class="bi bi-eye me-1"></i>
+                    Live
+                </span>
+                break;
+        }
+
+    </div>
+</div>
+
+<div class="row property-img-row">
+    @if (Model.Images.Count > 0)
+    {
+        <div class="col-sm my-3">
+            <a href="#" class="property-img-lg placeholder-glow" data-bs-toggle="modal" data-bs-target="#imageModal">
+                <div class="placeholder w-100 h-100" id="image0Placeholder"></div>
+                <img src="@Model.Images[0].FileUrl" alt="@Model.Images[0].FileName" id="image0" style="display: none" onload="hidePlaceholder('#image0')">
+            </a>

```

Comment:
All these image links look like they could be extracted into a function with maybe 3 inputs, I think that may makeit slightly nicer :)